examples/complete-alb/main.tf

```diff
@@ -506,22 +433,21 @@ resource "null_resource" "download_package" {
 
 module "lambda_with_allowed_triggers" {
   source  = "terraform-aws-modules/lambda/aws"
-  version = "~> 5.0"
+  version = "~> 3.0"
```

Comment:
```suggestion
  version = "~> 6.0"
```

Reply:
fixed in https://github.com/terraform-aws-modules/terraform-aws-alb/pull/305/commits/522c07009fa3cc79200886990f3ab8ba1ac76231